components/driver/sdmmc_host.c

```diff
@@ -236,12 +260,20 @@ esp_err_t sdmmc_host_set_card_clk(int slot, uint32_t freq_khz)
             break;
     }
     sdmmc_host_set_clk_div(host_div);
-    sdmmc_host_clock_update_command(slot);
+    err = sdmmc_host_clock_update_command(slot);
+    if (err != ESP_OK) {
+        ESP_LOGE(TAG, "set clk div failed");
```

Comment:
Probably not necessary to log here, since it's not something that can happen in practice. (Clock update commands don't depend on the state of the Card Interface Unit, and as such shouldn't be blocked by the bus state.)

(Same comment for other sdmmc_host_clock_update_command error checks.)

Reply:
You must be mistaken?  In my issue, we get stuck in 

`I (5109) sdmmc_periph: sdmmc_host_clock_update_command() - SDMMC.clkena.cclk_enable &= ~BIT(slot);`

Reply:
Sorry, have overlooked that. In that case the fix might be a bit different than ignoring the fact that the clock update didn't happen. We might be missing to reset some part of the peripheral. I'll check it and come back with more details.

Reply:
Yes would be great to fix the underlying issue, in addition to having reasonable timeouts.

Reply:
lmk how things go, Ivan. Thanks.